test/src/sdk-harness/test_projects/exponentiation/mod.rs

```diff
@@ -0,0 +1,78 @@
+use test_helpers::script_runner;
+use fuel_tx::ContractId;
+use fuels::prelude::*;
+use fuels_abigen_macro::abigen;
+use fuels_signers::wallet::Wallet;
+
+abigen!(
+    TestPowContract,
+    "test_artifacts/pow/out/debug/pow-abi.json"
+);
+
+#[tokio::test]
+async fn can_perform_exponentiation() {
+    let path_to_bin = "test_projects/exponentiation/out/debug/exponentiation.bin";
+    let return_val = script_runner(path_to_bin).await;
+    assert_eq!(return_val, 1);
+}
+
+#[tokio::test]
+#[should_panic(expected = "ArithmeticOverflow")]
+async fn overflowing_pow_u64_panics() {
+    let wallet = launch_provider_and_get_wallet().await;
+    let (pow_instance, _) = get_pow_test_instance(wallet).await;
+    let result = pow_instance.u64_overflow(100u64, 100u64)
+        .call()
+        .await
+        .unwrap();
+    dbg!(&result);
+
+    // assert_eq!(result.value, 100);
```

Comment:
Remove?

Reply:
yes

Reply:
Fixed in [e8ba47e](https://github.com/FuelLabs/sway/pull/1744/commits/e8ba47e5ff056a586ee2a6fb9f0c7979f87de88c)